doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -263,6 +269,8 @@ The service implements functionalities required by the `Fast Pair Procedure`_.
 The procedure is initiated by the Fast Pair Seeker after Bluetooth LE connection is established.
 No application interaction is required.
 
+In the Fast Pair subsystem disabled state, GATT operations on the Fast Pair service are rejected.
```

Comment:
The last small improvement (would be good to explain why it's implemented like that I think)

```
The Fast Pair GATT Service is statically defined, so it's still present in GATT database after Fast Pair is disabled. In the Fast Pair subsystem disabled state, GATT operations on the Fast Pair service are rejected.
```

Reply:
Addressed.